runtime-common/stdlib/string/string-context.h

```diff
@@ -30,13 +32,16 @@ inline constexpr std::string_view PERCENT_ = "%";
 
 }; // namespace string_context_impl_
 
-class StringLibContext final : vk::not_copyable {
+struct StringLibContext final : vk::not_copyable {
+  static constexpr int32_t STATIC_BUFFER_LENGTH = 1U << 23U;
+
+private:
   static constexpr int32_t MASK_BUFFER_LENGTH = 256;
 
-public:
-  static constexpr int32_t STATIC_BUFFER_LENGTH = 1U << 23U;
+  std::aligned_storage<STATIC_BUFFER_LENGTH + 1> storage;
```

Comment:
Are you sure that non-zeroing this memory won't lead to unexpected errors?
Did you test it with vkcom? This change seems hazardous
We have this kind of errors previously, see https://github.com/VKCOM/kphp/pull/1011/files#diff-a72cbbf809305fc21fdf70974cef5d9c5263b5ccfcb3d471fad561e03d323a0eR29

Reply:
By the way, we can align storage by size of cache line and it may lead benefits for small (up to 64 bytes) strings. I think it's worth to do, but up to you